pkg/products/threescale/reconciler.go

```diff
@@ -675,7 +675,7 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, serverClient k8scl
 			"threescale_component_element": "zync-que",
 		})
 
-		if r.installation.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManaged) {
+		if integreatlyv1alpha1.IsRHOAM(integreatlyv1alpha1.InstallationType(r.installation.Spec.Type)) {
```

Comment:
So we only want this set for RHOAM installs and not RHMI ? :thinking:

Reply:
IT should actually be just RHMI